python-shebang/hello.py

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env python
```

Comment:
You are very strict in your note about using `python` in the shebang:

> Never use the bare python command in a shebang, which may map to either `python2` or `python3` depending on the operating system and its configuration. The only exception would be if you wrote the script in a backward-compatible way and wanted it to run through both Python versions.

So I’d recommend using `python3` here, and maybe add another file named `hello_py2andpy3.py` (or something like that) with the bare `python` command.